web/src/reducers/dataReducer.js

```diff
@@ -40,7 +40,7 @@ const reducer = createReducer(initialState, (builder) => {
         }
         // Assign all data
         Object.keys(value).forEach((k) => {
-          state.exchanges[k] = value[k];
+          exchange[k] = value[k];
```

Comment:
I'm not sure this is correct, but I fix it later in my PR regardless

Reply:
It is, I actually screwed up on the refactor we did together - it should be `state.exchanges[key][k] = value[k]` and not `state.exchanges[k] = value[k]` :)